src/composables/common/useStatusList.ts

```diff
@@ -21,6 +22,10 @@ export function useStatusList (props, context) {
     // getting status list mouint and setting inside store
     // will make call once till page refresh
     getRoutingSlipStatusList()
+
+    routingSlipStatusList.value = routingSlipStatusList.value.filter(
+      status => status.code !== SlipStatus.REFUNDREJECTED
+    )
```

Comment:
Remove this please. This is just duplicating code that should already have executed.

I know you put this in to get the test working, but if you debug a little further you'll see you can make it work correctly without this by changing your test setup a bit